src/dist/js/search.js

```diff
@@ -1,5 +1,9 @@
 const formElement = document.getElementById('searchForm');
 formElement.addEventListener('submit', searchSubmit);
+
+const advSearchFormElement = document.getElementById('advSearchForm');
+advSearchFormElement.addEventListener('submit', searchSubmit);
+
 const resetButtonElement = document
```

Comment:
Lets update this one here too to be consistent with your current format

Reply:
@DarylCheung This is updated